### ISSUE TITLE
ENH: Add scale for divergence/mutations

### ISSUE THREAD

**Issue by corneliusroemer** [theosanderson/taxonium#268]
I realized that there doesn't seem to be a divergence/mutation scale in taxonium at the moment.

This would be incredibly useful, in the same way that maps show scales.

I'd probably display both, mutations and divergence. One could in principle add vertical gridlines, too, the way that Auspice does, in addition to a scale like ` |----5 muts---| --- 10 muts ---|`

**Comment by theosanderson** [comment]
Yep, dupe of https://github.com/theosanderson/taxonium/issues/86 (but no reason that should have been clear to you) thanks

**Comment by corneliusroemer** [comment]
I've just had the same thought again - this type I even made a little sketch.

Now I'd also suggest adding a y-axis scale bar. A student in our lab asked me whether there was really 1 dot per sequence in Taxonium. We played around a bit with it and realized how ridiculously many sequences there are per cm.

For 10M sequences, there's 1 sequence per 10 nanometers vertically 😄 

<img width="901" alt="Taxonium 2024-07-09 18 03 28" src="https://github.com/theosanderson/taxonium/assets/25161793/d9dbaa4a-1ba4-4239-af9f-9c8f2eba38af">

**Comment by theosanderson** [comment]
>  was really 1 dot per sequence in Taxonium

As an aside: the idea is that actually the view should be fairly indistinguishable to that if we *did* plot one point per sequence. (Until you zoom quickly and don't wait long enough for it to show the new zoom level)

**Comment by theosanderson** [comment]
(your stat is fun!)
